paper/paper.md

```diff
@@ -24,9 +24,9 @@ bibliography: paper.bib
+Magnetic data is one of the most prevelant types of geophysical datasets available on the surface of the Earth. The Curie depth is the depth at which rocks lose their magnetism. The most prevalent magnetic mineral is magnetite, which has a Curie point of 580°C, thus the Curie depth is often interpreted as the 580°C isotherm. Current methods to derive the Curie depth first compute the (fast) Fourier transform over a square window of a magnetic anomaly that has been reduced to the pole. The depth and thickness of magnetic sources is estimated from the slope of the radial power spectrum.
 
-Our Python package, `pycurious`, ingests maps of the Earth's magnetic anomaly and distributes the computation of Curie depth across multiple CPUs. `pycurious` implements the [@Tanaka1999] and [@Bouligand2009] methods for computing the thickness of a buried magnetic source. The former selects portions of the radial power spectrum in the low and high frequency domain to compute the depth of magnetic sources, while the latter fits an analytical solution to the entire power spectrum. We cast the [@Bouligand2009] method within a Bayesian framwork to estimate the uncertainty of Curie depth calculations. Common computational workflows and geospatial manipulation of magnetic data are covered in the Jupyter notebooks bundled with this package. The `mapping` module includes a set of functions that help to wrangle maps of the magnetic anomaly into a useable form for `pycurious`. Such an approach is commonly encountered for transforming global compilations of the magnetic anomaly, e.g. EMAG2 [@Meyer2017], from latitudinal/longitudinal coordinates to a local projection in eastings/northings.
+Our Python package, `pycurious`, ingests a maps of the Earth's magnetic anomaly and distributes the computation of Curie depth across multiple CPUs. `pycurious` implements the [@Tanaka1999] and [@Bouligand2009] methods for computing the thickness of a buried magnetic source. The former selects portions of the radial power spectrum in the low and high frequency domain to compute the depth of magnetic sources, while the latter fits an analytical solution to the entire power spectrum. We cast the [@Bouligand2009] method within a Bayesian framwork to estimate the uncertainty of Curie depth calculations. Common computational workflows and geospatial manipulation of magnetic data are covered in the Jupyter notebooks bundled with this package. The `mapping` module includes a set of functions that help to wrangle maps of the magnetic anomaly into a useable form for `pycurious`. Such an approach is commonly encountered for transforming global compilations of the magnetic anomaly, e.g. EMAG2 [@Meyer2017], from latitudinal/longitudinal coordinates to a local projection in eastings/northings.
```

Comment:
```ingests a maps of the Earth's magnetic anomaly```

is a bit unclear, by "magnetic anomaly" are you referring to a map which has had the IGRF removed? Would you mind clarifying this?

Reply:
I believe the phrase "magnetic anomaly" implicitly means the total magnetic field less the core and external field contributions:

__anomaly field = observed field - (core field + external fields)__

This has since been clarified in the text.